rules/groups-in-microsoft-365/rule.md

```diff
@@ -23,6 +23,10 @@ Microsoft provides a collaboration tool called Microsoft 365 email groups on the
 
 It is important to understand the differences between on-premises groups and Microsoft 365 groups.
 
+**NOTE**
```

Comment:
Hi @lloydcollinsssw 

We want to keep things consistent...

- [ ] 1. Please change to `**Note:** {{ TEXT }}`